### PR TITLE
To be merged Nov 17th - release(jakarta): release device-services

### PR DESCRIPTION
**Pull request by bill-mahoney**
Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

### CODE REVIEW

File: release/device-modbus-go.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: 'device-modbus-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '558c40fbdebbe4b00a0283e0ecd9585407ec2bc6'
```
Comment (lenny-intel) [inline release/device-modbus-go.yaml:7]:
```suggestion
commitId: '3366fcb7b8cfa87a3d941489b4b63a155443012c'
```
This is the final commit for the release.

File: release/device-mqtt-go.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: 'device-mqtt-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: 'f142b7d4d9dc0d163bb8d716c4eb7ddee57de476'
```
Comment (lenny-intel) [inline release/device-mqtt-go.yaml:7]: Waiting on last issue to be implemented and merged. 
https://github.com/edgexfoundry/device-mqtt-go/issues/321

File: release/device-rest-go.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+name: 'device-rest-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '09739088bb3ccddf347dd8b212f45a4687ee065f'
```
Comment (lenny-intel) [inline release/device-rest-go.yaml:7]:
```suggestion
commitId: '7759c41568adf15208c0dd5cadcdb8f87895acd8'
```
This is the final commit for the release.

File: release/device-snmp-go.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+name: 'device-snmp-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '8cef9076ddadae0bb4a2071504c212bd6ac7a64c'
```
Comment (lenny-intel) [inline release/device-snmp-go.yaml:7]:
```suggestion
commitId: 'd6a2d4438fc175be031f77f972aa414c83442272'
```
This is the final commit for the release.

File: release/device-virtual-go.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+name: 'device-virtual-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: 'ce5f781da4ac4c2a0d7531ae22c293687cd2d279'
```
Comment (lenny-intel) [inline release/device-virtual-go.yaml:7]:
```suggestion
commitId: 'd6a2d4438fc175be031f77f972aa414c83442272'
```

This is the final commit for the release.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: release/device-modbus-go.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+name: 'device-modbus-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '3366fcb7b8cfa87a3d941489b4b63a155443012c'
```
Comment (lenny-intel) [inline release/device-modbus-go.yaml:7]:
```suggestion
commitId: '494e11ba379638b0cea34cac06412dc20970075c'
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]
We should also include Device Camera, but with LTS=false

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by iain-anderson — APPROVED** [review]
LGTM

**Pull request merged**